src/server/api/controllers/vesselReport.controller.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
```

Comment:
Could you rename the file to `vessel-report.controller.js`. We use dashes for filenames and urls and camelCase in javascript.

Reply:
Renamed the file name from vesselReport.controller.js to vessel-report.controller.js